src/util/Options.js

```diff
@@ -123,6 +126,7 @@ class Options extends null {
       restSweepInterval: 60,
       failIfNotExists: true,
       userAgentSuffix: [],
+      supportFriendInvites: false,
```

Comment:
If we want to move forward with feature flags, we should probably group them under one object. Thoughts @discordjs/the-big-3?

Reply:
If we are going that way, grouping them would in my opinion be the best way to go about this.

Reply:
what should it be called? `features`?